src/components/DropdownStyles.css

```diff
@@ -37,6 +35,20 @@
   border-color: #006d7d;
 }
 
+/* Dropdown Content for Price and Date (Hidden by Default) */
+.dropdown-content-two-items {
+  margin-top: 18px;
+  height: 85px;
+  width: 196px;
+  display: none;
+  border-radius: 10px;
+  position: inherit;
```

Comment:
Let's change this to absolute so that the menu doesn't shift the things around it. Do the same for three-item